index.html

```diff
@@ -36,8 +36,8 @@
                 <a href="#advanced-auth">Authentication and Authorization</a>
               </h3>
               <p>
-                <span>Oct. 6th 2016</span>
-                <span>10:00 - 16:00 CET</span>
+                <span>Oct. 6th 2016</span><br>
```

Comment:
The `<span>`s should render as `block` already. If not we should fix that in the CSS instead of adding `<br>`s.